server/pbench/bin/pbench-audit-archive

```diff
@@ -0,0 +1,196 @@
+        if [ -s ${directories}.unexpected ]; then
+            printf "\t* Unexpected state directories found in ${controller} directory:\n"
+            printf "\t  ++++++++++\n"
+            cat ${directories}.unexpected
+            printf "\t  ----------\n"
+            let cnt=cnt+1
+        fi
+    else
+        printf "\t* No state directories found in ${controller} directory.\n"
+        let cnt=cnt+1
+    fi
+    rm -f ${directories} ${directories}.linkdirs ${directories}.unexpected
+}
+
+function verify_tarball_names {
+    controller=${1}
+
+    let cnt=0
+
+    find ${controller} -maxdepth 1 -type l -ls > ${unexpectedsymlinks} 2>&1
+    if [ -s ${unexpectedsymlinks} ]; then
+        printf "\t* Unexpected symlinks in ${controller} directory:\n"
+        printf "\t  ++++++++++\n"
+        cat ${unexpectedsymlinks}
+        printf "\t  ----------\n"
+        let cnt=cnt+1
+    fi
+    rm -f ${unexpectedsymlinks}
+
+    find ${controller} -maxdepth 1 -type f ! -name '*.tar.xz.md5' ! -name '*.tar.xz' > ${unexpectedfiles} 2>&1
+    if [ -s ${unexpectedfiles} ]; then
+        printf "\t* Unexpected files in ${controller} directory:\n"
+        printf "\t  ++++++++++\n"
+        cat ${unexpectedfiles}
+        printf "\t  ----------\n"
+        let cnt=cnt+1
+    fi
+    rm -f ${unexpectedfiles}
+
+    find ${controller} -maxdepth 1 -type f \( -name '*.tar.xz.md5' -o -name '*.tar.xz' \) > ${tarballs} 2>&1
+    if [ ! -s ${tarballs} ]; then
+        printf "\t* No files found in ${controller} directory\n"
+        let cnt=cnt+1
+    fi
+    rm -f ${tarballs}
+
+    return $cnt
+}
+
+function verify_prefixes {
+    controller=${1}
+    if [ ! -e ${controller}/.prefix ]; then
+        return 0
+    fi
+    if [ ! -d ${controller}/.prefix ]; then
+        printf "\t* Prefix directory, ${controller}/.prefix, is not a directory\n"
+        return 1
+    fi
+
+    let cnt=0
+
+    find ${controller}/.prefix -maxdepth 1 ! -name 'prefix.*' ! -name '*.prefix' > ${nonprefixes} 2>&1
+    if [ -s ${nonprefixes} ]; then
+        printf "\t* Unexpected file system objects in ${controller}.prefix directory:\n"
+        printf "\t  ++++++++++\n"
+        cat ${nonprefixes}
+        printf "\t  ----------\n"
+        let cnt=cnt+1
+    fi
+    rm -f ${nonprefixes}
+
+    find ${controller}/.prefix -maxdepth 1 -name 'prefix.*' > ${wrongprefixes} 2>&1
+    if [ -s ${wrongprefixes} ]; then
+        printf "\t* Wrong prefix file names found in ${controller}.prefix directory:\n"
+        printf "\t  ++++++++++\n"
+        cat ${wrongprefixes}
+        printf "\t  ----------\n"
+        let cnt=cnt+1
+    fi
+    rm -f ${wrongprefixes}
+
+    return $cnt
+}
+
+log_init $(basename $0) $LOGSDIR/$(basename $0)
+
+echo "start-$(timestamp)"
+
+# Initialize index mail content
+> $index_content
+
+# Construct the report file
+> $report
+
+find $ARCHIVE -maxdepth 1 ! -type d -ls | sort -k 11 > $badcontrollers
```

Comment:
Can you add a comment to explain the 11? Maybe even the whole command: "find non-directories in the archive top-level directory and sort them alphabetically by name"  - assuming that's correct.